README.md

```diff
@@ -75,18 +75,27 @@ python3 -m manim example_scenes.py SquareToCircle -pl
 Since it's a bit tricky to get all the dependencies set up just right, there is a Dockerfile and Compose file provided in this repo as well as [a premade image on Docker Hub](https://hub.docker.com/r/eulertour/manim/tags/). The Dockerfile contains instructions on how to build a manim image, while the Compose file contains instructions on how to run the image.
 
 In order to do this with the Compose file, you must set the `INPUT_PATH`
-environment variable to the directory containing your source code and the
-`OUTPUT_DIRECTORY` environment variable to the directory where you want media
-to be written.
+environment variable to the directory containing manim repository and the
```

Comment:
Manim is now stored inside the docker image, so `INPUT_PATH` should be the absolute path to the directory containing the source code.